### PR TITLE
feat: Landing-page and routing added

### PR DESCRIPTION
**Pull request by sumitch29**
- Landing page added
- Routing added

### CODE REVIEW

**Review by TusharJade — APPROVED** [review]
The code is clean. Just see the images in the product listing page are broken just fix it and in some input tags you use unnecessary attributes either delete that or gave it proper naming, other than that everything looks good, good to go for merged 👍

File: src/components/Filter/Filter.jsx [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+            </div>
+            <h2>Price</h2>
+            <div class="input-range center">
+              <input
+                type="range"
+                min="10"
+                max="1000"
+                value="500"
+                name=""
+                id=""
```
Comment (vikrantkode) [inline src/components/Filter/Filter.jsx]: Try not to leave any unused declared variable.

File: src/components/Filter/filter.css [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+  grid-template-columns: 1fr 5fr;
+}
+
+.body {
+  background-color: var(--background-color);
+}
+
+.header-container {
+  box-shadow: 1px 5px 5px gray;
+}
```
Comment (vikrantkode) [inline src/components/Filter/filter.css]: dont hardcode the colours. Use variables

File: src/components/Filter/filter.css [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+.filter-box {
+  padding: 2rem 2rem;
+  width: 23rem;
+  height: 65rem;
+  background-color: var(--background-color);
+  box-shadow: 5px 5px 9px 2px grey;
+}
```
Comment (vikrantkode) [inline src/components/Filter/filter.css]: dont hardcode the colours. Use variables

File: src/components/Navbar/Navbar.jsx [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+              Store
+            </Link>
+          </p>
+        </div>
+        <div className="search-box center">
+          <input
+            className="padding-5 search"
+            type="text"
+            name=""
+            id=""
```
Comment (vikrantkode) [inline src/components/Navbar/Navbar.jsx]: Try giving proper values for better readibility.

**Review by vikrantkode — APPROVED** [review]
- Some Suggestions given. You can have a look on them while proceding.

**Pull request merged**